DEVELOP.rst

```diff
@@ -0,0 +1,126 @@
+Run locally
+###########
+
+
+#. Install Python 3.8+ and create a virtual environment.
+
+   .. code-block::
+  
+      python -m venv my_venv
+  
```

Comment:
I assume that this is here to guide those not very familiar with Python's virtual environments, right? I mean, all those familiar can be simply addressed by "setup your Python virtual environment, then within it...". Maybe I'm wrong on this point, but I rather suspect that people coming from JS might not think of setting up Python environment as a trivial and self-explanatory rote action, even if they are otherwise technically adept and competent.

So either a link to the python venv docs is needed (frankly, too much text and the basics are hard to figure out) or just more explicit steps need to be given.
The step above will *create* a venv, but you also need to activate it to work within it. So that should be spelled out:
- To enter the virtual environment:
   ```
   $ source env/bin/activate
   (env) $
   ```
- To exit the virtual environment:
   ```
   (env) $ deactivate
   ```
The rest of the work is assumed to happen while the virtual environment is activated. It may be worthwhile to state this explicitly as well.